Gemfile

```diff
@@ -11,6 +11,12 @@ gemspec
 # checkout the latest version (develop) from github.
 allow_local = ENV['FAVOR_LOCAL_GEMS']
 
+# uncomment when you want CI to use develop branch of extension gem
+gem 'openstudio-extension', github: 'NREL/OpenStudio-extension-gem', branch: 'develop'
```

Comment:
should these be uncommented?

Reply:
Yes, they should stay un-commented for now, we could comment it out after the 3.2 release until we have fixes in extension gem or standards gem that are not in 3.2 OpenStudio installer.